### ISSUE TITLE
SyntaxError while installing nodes

### ISSUE THREAD

**Issue by MrTreffnix601** [Hilscher/node-red-contrib-s7comm#5]
I get following Error when installing s7comm..

15 Oct 08:19:05 - [info] Installing module: node-red-contrib-s7comm
15 Oct 08:20:31 - [info] Installed module: node-red-contrib-s7comm
15 Oct 08:20:32 - [info] Added node types:
15 Oct 08:20:32 - [info]  - node-red-contrib-s7comm:s7comm read : SyntaxError: Unexpected token )
15 Oct 08:20:32 - [info]  - node-red-contrib-s7comm:s7comm write : SyntaxError: Unexpected token )
15 Oct 08:20:32 - [info]  - node-red-contrib-s7comm:s7comm : SyntaxError: Unexpected token )

When I add some nodes from this palette, I get this error..

15 Oct 08:24:26 - [info] Waiting for missing types to be registered:
15 Oct 08:24:26 - [info]  - s7comm read
15 Oct 08:24:26 - [info]  - s7comm write
15 Oct 08:24:45 - [info] Stopping flows
15 Oct 08:24:45 - [info] Stopped flows
15 Oct 08:24:45 - [info] Waiting for missing types to be registered:
15 Oct 08:24:45 - [info]  - s7comm
15 Oct 08:24:45 - [info]  - s7comm read
15 Oct 08:24:45 - [info]  - s7comm write

**Comment by sembaye** [comment]
Hi,
Can you give me more in formations? (OS, Which version was installed before ...)
I could install s7comm on a clean system without any Error. On my Ubuntu as well as on Windows.
FYI: Since V.1.6 there's a major change. I import the lib net-keepalive which is OS dependent.

**Comment by sembaye** [comment]
#4

**Comment by MrTreffnix601** [comment]
Hello,
I´m using Yocto Linux on the Siemens IoT Gateway 2040. 
node.js version: v6.12.3
Node-RED version: v0.16.2

I try to use s7comm in version 1.1.6

**Comment by konradheidrich** [comment]
I tested again, this time with exact the same versions of node.js and Node-RED as @MrTreffnix601 mentioned above - with the same error result as he had.
For further tests I updated node.js to v8.12.0 (no other versions between tested) and restarted Node-RED. It works fine.
Not yet sure, what the trigger is, but we are using quite new node.js and Node-RED version, therefore we did not recognize this behavior in our environment.
